lib/aws-ec2/bootstrap/chart_values/external-dns.j2.yaml

```diff
@@ -0,0 +1,22 @@
+provider: {{ external_dns_provider }}
+
+{% if external_dns_provider == "cloudflare" %}
+cloudflare:
+  apiToken: "{{ cloudflare_api_token }}"
+  email: "{{ cloudflare_email }}"
+  proxied: {{ cloudflare_proxied|default(value="false") }}
+{% elif external_dns_provider == "pdns" %}
+# Qovery DNS
+pdns:
+  apiUrl: "{{ qoverydns_api_url }}"
+  apiPort: "{{ qoverydns_api_port }}"
+  apiKey: "{{ qoverydns_api_key }}"
```

Comment:
Seems to be an incoherence with those `qoverydns_` values ?
Here in github there is `qoverydns_` whereas in [gitlab](https://gitlab.com/qovery/engine/qovery-engine/-/merge_requests/255/diffs#77de5415e2e22e2591cd7f70c7895e5c5ff25eac_413_432) part there is `qovery_dns_`

Reply:
updating gitlab side